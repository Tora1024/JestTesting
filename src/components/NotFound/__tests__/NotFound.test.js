import React from 'react';
import { shallow } from 'enzyme';
import ComponentUnderTest from '../NotFound';

describe('components/NotFound/NotFound.js', () => {
  let wrapper;

  describe('in general', () => {
    beforeEach(() => {
      wrapper = shallow(<ComponentUnderTest />);
    });
    it('should render correct snapshot', () => {
      expect(wrapper).toMatchSnapshot();
    });
  });
});
