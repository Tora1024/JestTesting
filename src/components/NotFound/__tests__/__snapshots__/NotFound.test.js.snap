// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/NotFound/NotFound.js in general should render correct snapshot 1`] = `
<main
  className="container"
>
  <section
    className="inner-container"
  >
    <div
      className="lost"
    />
    <h1
      className="headline"
    >
      Uhh ohh! Not sure what happened, but you are out of reach right now..
    </h1>
    <Link
      className="go-back"
      replace={false}
      to="/"
    >
      Please, take me back home!
    </Link>
  </section>
</main>
`;
